### ISSUE TITLE
JSON parsing fails with negative integers

### ISSUE THREAD

**Issue by GoogleCodeExporter** [kmmbvnr/common-schema#46]
```
Thanks for common schema, it's pretty cool.

Using 2.1 JSON parsing fails with negative integers:

mysql> select common_schema.json_to_xml('{"foo":1}'); 
+----------------------------------------+
| common_schema.json_to_xml('{"foo":1}') |
+----------------------------------------+
| <foo>1</foo>                           |
+----------------------------------------+
1 row in set (0.03 sec)

mysql> select common_schema.json_to_xml('{"foo":-1}'); 
+------------------------------------------+
| common_schema.json_to_xml('{"foo":-1}') |
+------------------------------------------+
| NULL                                     |
+------------------------------------------+
1 row in set (0.03 sec)

Thanks!
```

Original issue reported on code.google.com by `martin.s...@gmail.com` on 26 Jul 2013 at 11:51

**Comment by GoogleCodeExporter** [comment]
```
This issue was closed by revision r504.
```

Original comment by `shlomi.n...@gmail.com` on 26 Jul 2013 at 12:42
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Fix will be released in 2.1.1, in a couple weeks. Until then, you may import 
_get_json_token.sql onto your existing install.
```

Original comment by `shlomi.n...@gmail.com` on 26 Jul 2013 at 12:43

**Comment by GoogleCodeExporter** [comment]
```
thank you, works perfect.
```

Original comment by `martin.s...@gmail.com` on 26 Jul 2013 at 1:13
